frontend/rust-lib/flowy-user/src/services/user_session.rs

```diff
@@ -288,28 +289,22 @@ impl UserSession {
   fn set_session(&self, session: Option<Session>) -> Result<(), FlowyError> {
     tracing::debug!("Set user session: {:?}", session);
     match &session {
-      None => KV::remove(&self.session_config.session_cache_key)
-        .map_err(|e| FlowyError::new(ErrorCode::Internal, &e))?,
-      Some(session) => KV::set_str(
-        &self.session_config.session_cache_key,
-        session.clone().into(),
-      ),
+      None => KV::remove(&self.session_config.session_cache_key),
+      Some(session) => {
+        KV::set_object(&self.session_config.session_cache_key, session.clone())
+          .map_err(internal_error)?;
+      },
     }
     Ok(())
   }
 
   fn get_session(&self) -> Result<Session, FlowyError> {
-    match KV::get_str(&self.session_config.session_cache_key) {
+    match KV::get_object::<Session>(&self.session_config.session_cache_key) {
       None => Err(FlowyError::unauthorized()),
-      Some(s) => Ok(Session::from(s)),
+      Some(session) => Ok(session),
```

Comment:
use `ok_or_else` would be better since the `Session::from` has been removed.

Reply:
I don't know how. Give me suggestions. I just want to one click